src/app/src/main/java/com/example/bookmark/util/FormValidator.java

```diff
@@ -0,0 +1,68 @@
+package com.example.bookmark.util;
+
+import android.widget.EditText;
+
+import com.google.android.material.textfield.TextInputLayout;
+
+public class FormValidator {
```

Comment:
Imo this class name should be only for the signup validation, therefore making it SignupFormValidator would perhaps be more clear. We can make other validator classes as needed for other purposes.

Reply:
I'm using these same functions for the "EditProfileActivity" but i suppose youre right

Reply:
Right, sorry. In general then I guess, a more specific name would be good.

Reply:
changed to UserInfoFormValidator